extensions/cometapi/src/translate.tsx

```diff
@@ -0,0 +1,143 @@
+import {
+  Action,
+  ActionPanel,
+  Detail,
+  Form,
+  Icon,
+  getSelectedText,
+  showToast,
+  Toast,
+  useNavigation,
+  getPreferenceValues,
+} from "@raycast/api";
+import { useEffect, useState } from "react";
+import { openai, getGlobalModel } from "./api";
+
+type Prefs = {
+  prompt_translate?: string;
+  model_translate?: string;
+  primary_language_translate?: string;
+  secondary_language_translate?: string;
+};
```

Comment:
@greptileai, you didn't mention that the user shouldn't add the Prefs interface/type themselves since it's already autogenerated in `raycast-env.d.ts`